actors/builtin/reward/reward_actor.go

```diff
@@ -49,10 +49,9 @@ func (a Actor) Constructor(rt vmr.Runtime, _ *adt.EmptyValue) *adt.EmptyValue {
 }
 
 type AwardBlockRewardParams struct {
-	Miner        addr.Address
-	Penalty      abi.TokenAmount // penalty for including bad messages in a block
-	GasReward    abi.TokenAmount // gas reward from all gas fees in a block
-	NominalPower abi.StoragePower
+	Miner     address.Address
+	Penalty   abi.TokenAmount // penalty for including bad messages in a block
+	GasReward abi.TokenAmount // gas reward from all gas fees in a block
```

Comment:
Just FYI, I believe we'll need to add a winning ticket count parameter here later.

Reply:
Ah, yeah. we will need that.